tests/torchtune/modules/low_precision/test_nf4_linear.py

```diff
@@ -59,9 +55,10 @@ def test_state_dict(self, dtype):
         assert isinstance(state_dict["weight"], NF4Tensor)
 
     @pytest.mark.parametrize("dtype", [torch.bfloat16, torch.float32])
-    def test_output_dtype(self, dtype):
+    @pytest.mark.parametrize("bias", [True, False])
```

Comment:
What is the point of adding bias to this test? The dtype isn't changing and you're only checking the dtype?

Reply:
Agreed it's pretty trivial but I'd like to at least build `FrozenNF4Linear` with bias somewhere in our unit tests, and the overhead of this unit test is tiny